package.json

```diff
@@ -39,7 +39,7 @@
     "cypress": "^7.6.0",
     "if-env": "^1.0.4",
     "jest": "^24.9.0",
-    "mocha": "^5.2.0",
+    "mocha": "^9.1.0",
```

Comment:
No breaking changes that affect us?

Reply:
tests run properly, I don't see any other area where this package could have an impact